docs/user/device-api-statuses.md

```diff
@@ -8,13 +8,15 @@ The Device Status represents the availability and health of the device's hardwar
-| `Online` | All hardware resources and operating system services are reported to be healthy. | `!deviceIsDisconnected && !deviceIsRebooting && ∀ r∈{CPU, Memory, Disk}, status.resources[r]∈{Healthy}` |
-| `Degraded` | One or more hardware resources or operating system services are reported to be degraded but in a still functional or recovering state. | `!deviceIsDisconnected && !deviceIsRebooting && ∀ r∈{CPU, Memory, Disk}, status.resources[r]∉{Error, Critical} && ∃ r∈{CPU, Memory, Disk}, status.resources[r]∈{Degraded}` |
-| `Error` | One or more hardware resources or operating system services are reported to be in error or critical state. | `!deviceIsDisconnected && !deviceIsRebooting && ∃ r∈{CPU, Memory, Disk}, status.resources[r]∈{Error, Critical}` |
-| `Rebooting` | The device is rebooting. | `!deviceIsDisconnected && deviceIsRebooting` |
-| `Offline` | The device is disconnected from the service but may still be running. | `deviceIsDisconnected` |
+| Status              | Description                                                                                                                            | Formal Definition<sup>1</sub> |
+|---------------------|----------------------------------------------------------------------------------------------------------------------------------------| ----------------------------- |
+| `Online`            | All hardware resources and operating system services are reported to be healthy.                                                       | `!deviceIsDisconnected && !deviceIsRebooting && ∀ r∈{CPU, Memory, Disk}, status.resources[r]∈{Healthy}` |
+| `Degraded`          | One or more hardware resources or operating system services are reported to be degraded but in a still functional or recovering state. | `!deviceIsDisconnected && !deviceIsRebooting && ∀ r∈{CPU, Memory, Disk}, status.resources[r]∉{Error, Critical} && ∃ r∈{CPU, Memory, Disk}, status.resources[r]∈{Degraded}` |
+| `Error`             | One or more hardware resources or operating system services are reported to be in error or critical state.                             | `!deviceIsDisconnected && !deviceIsRebooting && ∃ r∈{CPU, Memory, Disk}, status.resources[r]∈{Error, Critical}` |
+| `Rebooting`         | The device is rebooting.                                                                                                               | `!deviceIsDisconnected && deviceIsRebooting` |
+| `Offline`           | The device is disconnected from the service but may still be running.                                                                  | `deviceIsDisconnected` |
+| `AwaitingReconnect` | The device is awaiting reconnection after a restore was made.                                                                          | `deviceIsDisconnected` |
+| `ConflictPaused`    | The device is paused because the device reported a newer renderedVersion than the one we have.   | `deviceIsDisconnected` |
```

Comment:
```suggestion
| `ConflictPaused`    | The device is paused because the device reported a renderedVersion not known to the service.   | `deviceIsDisconnected` |
```